packages/reason-parser/reason-parser.2.0.0/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "1.2"
+maintainer: "Jordan Walke <jordojw@gmail.com>"
+authors: "Jordan Walke <jordojw@gmail.com>"
+homepage: "https://github.com/facebook/reason"
+bug-reports: "https://github.com/facebook/reason/issues"
+license: "BSD"
```

Comment:
@cristianoc What's the problem with simply switching to what I suggest so that we get discriminating information when we mine opam files ? The current information is obviously a lie.